settings/sqltasks.setting.php

```diff
@@ -21,4 +21,14 @@
     'is_contact'  => 0,
     'description' => ts('Default configuration (ID) for new SQL Tasks.'),
   ],
+  'sqltasks_sftp_max_retries' => [
```

Comment:
We should add a separate `sqltasks_sftp_retry_initial_wait` setting so we have more control over the exponential retry behaviour.